index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>calculator</title>
+  </head>
+  <body>
+    <div>
+      <input type="text" id="screen" value="" currentOperation="" disabled />
+    </div>
+
+    <button id="clearButton">AC</button>
+    <button id="previousAnswerButton">PreviousAns</button>
+    <button id="getHistory">HIS</button>
+    <button id="operationButtons">/</button>
+
+    <br />
+    <button numbersToPress>1</button>
+    <button numbersToPress>2</button>
+    <button numbersToPress>3</button>
+    <button id="operationButtons">*</button>
+    <br />
+    <button numbersToPress>4</button>
+    <button numbersToPress>5</button>
+    <button numbersToPress>6</button>
+    <button id="operationButtons">+</button>
+    <br />
+    <button numbersToPress>7</button>
+    <button numbersToPress>8</button>
+    <button numbersToPress>9</button>
+    <button id="operationButtons">-</button>
+    <br />
+    <button numbersToPress>0</button>
+    <button id="equalsButton">=</button>
+    <button numbersToPress>.</button>
```

Comment:
Why is "." treated as a number button? The logic for "." is completely different to the logic for any of the other numbers

Reply:
Good point. I have Fixed that.